src/client.cpp

```diff
@@ -243,10 +243,14 @@ int waybar::Client::main(int argc, char *argv[]) {
   std::string config;
   std::string style;
   std::string bar_id;
+  std::string log_level;
   auto        cli = clara::detail::Help(show_help) |
              clara::detail::Opt(show_version)["-v"]["--version"]("Show version") |
              clara::detail::Opt(config, "config")["-c"]["--config"]("Config path") |
              clara::detail::Opt(style, "style")["-s"]["--style"]("Style path") |
+             clara::detail::Opt(
+                 log_level,
+                 "trace|debug|info|warning|error|critical|off")["-l"]["--log-level"]("Log level") |
```

Comment:
This doesn't look nice, but I haven't found a way to improve formatting without removing list of levels.
```
where options are:
  -?, -h, --help                            display usage information
  -v, --version                             Show version
  -c, --config <config>                     Config path
  -s, --style <style>                       Style path
  -l, --log-level <trace|debug|info         Log level
  |warning|error|critical|off>
  -b, --bar <id>                            Bar id
```